pyproject.toml

```diff
@@ -1,7 +1,73 @@
 [build-system]
-requires = ["setuptools >= 40.6.0", "wheel"]
+requires = ["setuptools >= 75.3.2", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "django-money"
+dynamic = ["version"]
+description = "Adds support for using money and currency fields in django models and forms. Uses py-moneyed as the money implementation."
+readme = "README.rst"
+license = { text = "BSD" }
+requires-python = ">=3.8"
+maintainers = [
+    { name = "Greg Reinbach", email = "greg@reinbach.com" },
+]
+keywords = [
+    "django",
+    "money",
+    "py-money",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Framework :: Django",
+    "Framework :: Django :: 4.2",
+    "Framework :: Django :: 5.0",
+    "Framework :: Django :: 5.1",
+    "Framework :: Django :: 5.2",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = [
+    "Django>=2.2",
+    "py-moneyed>=2.0,<3.1",
+    "setuptools",
```

Comment:
Is setuptools actually needed by the package or is it more of a build dependency?

Reply:
I don't think the package should depend on it. That might be an outdated thing. Does Git Blame show anything? Otherwise, let's get rid of it :+1:

Reply:
I've tracked it down all the way back to https://github.com/django-money/django-money/commit/c01d29b4b2839976fd457a1e950ed5800150b315 (2012!)

I can see how it might have been needed back then, but I'm fairly sure it's no longer the case with the concept of build backend...

Reply:
Agree!